templates/zerver/help/configure-notification-bot.md

```diff
@@ -1,21 +1,83 @@
-# Configure notification bot
+# Notification bot
+
+The Zulip notification bot automatically generates messages for
+various organization level events, including:
+
+* Stream settings changes (such as [name](/help/rename-a-stream),
+  [description](/help/change-the-stream-description),
+  [permission](/help/stream-permissions) and
+  [policy](/help/stream-sending-policy) updates)
+* A topic being [resolved/unresolved](/help/resolve-a-topic)
+* New stream and new user announcements
```

Comment:
Let's break the last one into separate bullets, and expand the stream one. Something like:

* New public stream announcements. Private streams are not announced.